Modules/Core/Common/src/itkObjectFactoryBase.cxx

```diff
@@ -107,23 +100,18 @@ class ObjectFactoryBasePrivate : public LightObject
   ~ObjectFactoryBasePrivate() override
   {
     itk::ObjectFactoryBase::UnRegisterAllFactories();
-    if (m_InternalFactories)
+    for (auto & m_InternalFactorie : m_InternalFactories)
```

Comment:
English spelling is `Factory`, not `Factorie` 😄

Reply:
Indeed @dzenanz The term `m_InternalFactorie` appears to be introduced with a code _refactorieng_  ;-) PR #1628 commit 7f2c44cf534ac8e4a77eb0ec111c84b1c5d34be7: https://github.com/InsightSoftwareConsortium/ITK/blob/7f2c44cf534ac8e4a77eb0ec111c84b1c5d34be7/Modules/Core/Common/src/itkObjectFactoryBase.cxx#L109-L112

Anyway, I'd rather not include fixes of old typos with this PR, though. These can be fixed later, IMhO. Unless you insist...

Reply:
My check with
```shell
cd git/ITK
egrep -Ri 'factorie([^s]|$)'
```
shows the misspelling "factorie" occurs only in this variable in this file, on exactly two lines of code.

Reply:
I thought you are introducing this typo in this commit, due to Dutch spelling being `Factorie` 😄

Of course, it can be fixed later. Maybe a new STYLE commit as part of this PR?

Reply:
@Leengit Thanks, but then you may also try `for (const auto & m_` for similar typo's. 😺  (Remember we use the `m_` prefix only for member variables.)  Anyway, I'd rather defer such typo fixes to another PR, someone else, possibly.... unless you click "Request changes"!

Reply:
FWIW,
```shell
egrep -Ri 'for ?\(auto ?& ?[a-z_]*ie[a-z_]* '
```
returns only the case we know about.

Reply:
OK, but `for (const auto & m_` does occur twice anyway.

Reply:
For the issue of whether the looping variable should properly begin with `m_`, I find a total of three instances with `egrep -Ri 'for ?\((const )?auto ?&? ?m_[a-z_]* ?'`:
```cpp
Modules/IO/GDCM/src/itkGDCMSeriesFileNames.cxx:    for (const auto & m_InputFileName : m_InputFileNames)
Modules/Core/Common/src/itkObjectFactoryBase.cxx:      for (auto & m_InternalFactorie : *m_InternalFactories)
Modules/Core/Mesh/include/itkConnectedRegionsMeshFilter.hxx:    for (const auto & m_RegionSize : m_RegionSizes)
```

Reply:
@Leengit Would you possibly like to rename those three looping variables for us?  😃

Reply:
I've created Pull Request https://github.com/InsightSoftwareConsortium/ITK/pull/3646.